appdev/guides/contenthub.rst

```diff
@@ -0,0 +1,157 @@
+=============================================
+The content hub. Tips and tricks
+=============================================
+.. figure:: /_static/images/appdev/guides/contenthubimages/01.png
+        :align: center
+
+        Content Hub Share Page
+
+On Ubuntu Touch the apps are confined. The way of sharing files between them is through the Content Hub, a part of the system that takes care of file import, export and sharing.
+
+Different ways of sharing the content
+-------------------------------------
+As we can see in the `documentation <https://api-docs.ubports.com/sdk/apps/qml/Ubuntu.Content/ContentHandler.html#detailed-description>`_, there are several ways of handling the file to be shared:
+
+- ContentHandler.Source (The app selected is going to be the source of the file imported)
+- ContentHandler.Destination (The app selected is going to be the destination of the file exported)
+- ContentHandler.Share (The app selected is going to be the app from witch the file is going to be shared from)
+
+Importing
+^^^^^^^^^
+.. figure:: /_static/images/appdev/guides/contenthubimages/02.png
+        :align: center
+
+        Webapp Creator on the OpenStore.io
+
+Looking into the code of Webapp Creator, we’ll find the code to import an image to be used as icon. Tapping on the place holder will open the Content Hub that will let us choose from where to import the image (`see the source code <https://gitlab.com/cibersheep/webapp-creator/blob/master/webapp-creator/app/ImportPage.qml#L38>`_)
+
+.. code:: qml
+
+        ContentPeerPicker {
+            anchors { fill: parent; topMargin: picker.header.height }
+            visible: parent.visible
+            showTitle: false
+            contentType: picker.contentType //ContentType.Pictures
+            handler: picker.handler //ContentHandler.Source
+
+**ContentPeerPicker** is the element that shows the apps.
+**contentType** is passed in the code as ContentType.Pictures. So, we will only see apps from which we only can import images.
+And **handler** is passed in the code as ContentHandler.Source. As we want to import an image from the app selected in the Content Hub.
+
+Exporting
+^^^^^^^^^
+.. figure:: /_static/images/appdev/guides/contenthubimages/04.png
+        :align: center
+
+        Gelek in the OpenStore.io
+
+In Gelek, we are going to end with some saved games that we want to save in our device or share with ourselves (in Telegram and then save them to our computer).
+
+.. image:: /_static/images/appdev/guides/contenthubimages/05.png
+
+Tapping on the download icon we will get a Content Hub to «save» the game file (which is actually an export).
+
+The game file is a file type *.glksave.* So we will tell to the Content Hub that is of type «All» (`see code <https://gitlab.com/cibersheep/gelek/blob/master/app/InstallPage.qml#L38>`_).
```

Comment:
In the case of file extensions, code, or other non-translatable strings, it's better to place them in double backquotes (\`). This will make them appear monospaced in the output (which makes them easier to read) and hopefully it shows translators that they shouldn't translate.